scripts/dist.sh

```diff
@@ -26,7 +26,7 @@ mkdir -p build/pkg
 
 # Do a hermetic build inside a Docker container.
 docker build -t tendermint/tendermint-builder scripts/tendermint-builder/
```

Comment:
We could be running the dist script in a docker container, rather than calling docker from within the dist script.

Reply:
That way, we don't rely on docker either.

Reply:
I'm all for this.

Reply:
Still needs to change?